scripts/scripts.js

```diff
@@ -981,6 +981,14 @@ export async function createBreadCrumbs(crumbData, chevronAll = false) {
   return breadcrumbContainer;
 }
 
+/**
+ * Logs the information about an error encountered by the site.
+ * @param {Error} e Error information to log.
+ */
+export async function captureError(e) {
+  sampleRUM('error', e);
```

Comment:
RUM works by specifying a `source` and `target` property in the second parameter. Typically, the `source` is the identifier for the data point and `target` is the value.
I'd probably go with:
```suggestion
  sampleRUM('error', { source: 'newletter-signup', target: e.message);
```